plugins/woocommerce-admin/client/marketplace/components/promotions/promotions.tsx

```diff
@@ -12,37 +14,21 @@ declare global {
 	}
 }
 
-type Promotion = {
-	date_from_gmt: string;
-	date_to_gmt: string;
-	format: string;
-	pages: Page[];
-	position: string;
-	content: { [ locale: string ]: string };
-	icon?: string;
-	is_dismissible?: boolean;
-	menu_item_id?: string;
-	style?: string;
-};
-
-type Page = {
-	page: string;
-	path: string;
-	tab?: string;
-};
-
-const Promotions: () => null | JSX.Element = () => {
+const Promotions: ( {
+	format,
+}: {
+	format?: string;
+} ) => null | JSX.Element = ( { format = 'notice' } ) => {
```

Comment:
Thank you for this commit! It's 💯.

My only proposal is to make the argument required.

```suggestion
	format: string;
} ) => null | JSX.Element = ( { format } ) => {
```

Reply:
Thanks Boro, committed.